app/assets/javascripts/components/article_finder/article_finder.jsx

```diff
@@ -73,22 +80,22 @@ const ArticleFinder = createReactClass({
         value_key="search_term"
         required
         editable
-        label={I18n.t('article_finder.category')}
-        placeholder={I18n.t('article_finder.category')}
+        label="Search"
```

Comment:
These changed interface messages should be done in the locale files.

Reply:
Done!